src/ReinforcementLearningCore/src/core/stop_conditions.jl

```diff
@@ -167,7 +167,7 @@ Return `true` if the environment is terminated.
 """
 struct StopWhenDone <: AbstractStopCondition end
 
-check_stop(s::StopWhenDone, agent, env) = is_terminated(env)
+check!(s::StopWhenDone, agent, env) = is_terminated(env)
```

Comment:
`StopIfEnvTerminated`, since it does exactly the same check as the reset condition.